krkn/utils/VirtChecker.py

```diff
@@ -199,8 +210,10 @@ def run_virt_check(self, vm_list_batch, virt_check_telemetry_queue: queue.Queue)
                     virt_check_telemetry.append(VirtCheck(virt_check_tracker[vm]))
             else:
                 virt_check_telemetry.append(VirtCheck(virt_check_tracker[vm]))
-        virt_check_telemetry_queue.put(virt_check_telemetry)
-    
+        try:
+            virt_check_telemetry_queue.put(virt_check_telemetry)
+        except Exception as e:
```

Comment:
if this queue is a bounded queue (with a maximum item number) and it gets filled before being consumed, the `put` method might wait forever causing a deadlock I'd suggest to switch the type hint from the `queue.Queue` `queue.SimpleQueue` to clarify this aspect.